core/src/main/resources/error/error-classes.json

```diff
@@ -4298,7 +4298,7 @@
   },
   "_LEGACY_ERROR_TEMP_2175" : {
```

Comment:
This special error class name means it's not a public error. This is just a temporary name to speed up error migration. Eventually it will be either removed if it's indeed an internal error, or renamed to an offical name.